packages/e2e-tests/testFixtureCloneRecording.ts

```diff
@@ -23,19 +30,27 @@ const testWithCloneRecording = base.extend<TestIsolatedRecordingFixture>({
       throw new Error("Invalid recording");
     }
 
-    if (process.env.REPLAY_DISABLE_CLONE) {
-      await use({
-        page,
-        recordingId: exampleRecordings[exampleKey].recording,
-      });
-      return;
-    }
-
     let newRecordingId: string | undefined = undefined;
     try {
       const { recording } = exampleRecordings[exampleKey];
+      console.log("       Cloning recording");
       newRecordingId = await cloneTestRecording(recording);
 
+      const callbacks: any = {
+        onClose: console.error,
+        onError: console.error,
+      };
+      const client = new SimpleProtocolClient(new WebSocket(DISPATCH_URL), callbacks, console.log);
+
+      try {
+        console.log("       Processing recording");
+        await client.sendCommand("Recording.processRecording", {
+          recordingId: newRecordingId,
+        });
+      } catch (e) {
+        console.warn("       Error processing recording; ignoring.");
+      }
```

Comment:
Can we re-use the code we already have for this (`loadRecording`)?

Reply:
Yup--done.  (I was worried about the little console-animation waiter thingy messing w/ playwright, but it seems fine.)